doc/md/cloud/deployment.mdx

```diff
@@ -48,22 +42,17 @@ atlas {
   }
 }
```

Comment:
`cloud token` is not block needed. Change it to: `ATLAS_TOKEN` env variable instead.